docs/userman/advanced_features/auth/authentication.md

```diff
@@ -427,3 +427,73 @@ connect to the directory service:
         # Returns a valid Credentials instance if the LDAP server validates the password
         # "pass123" for user "user1". Raises an InvalidCredentials exception otherwise.
         ```
+
+# Microsoft Entra ID Authenticator
+
+Taipy also provides support for Microsoft Entra ID authentication.
+
+An authenticator using the *Entra ID* protocol has two specific parameters that need to be provided
+in order to properly connect to the Microsoft Entra ID service:
+
+- *client_id*: The client ID of the Entra ID application. The application must be registered in the
+    Azure Entra ID portal and have the required permissions including the "User.Read" and
+    "GroupMember.Read.All" permissions.
+- *tenant_id*: The tenant ID of the Entra ID organization.
+
+??? note "Entra ID application management"
+
+    Using the Entra ID authentication protocol assumes that an Entra ID application is already set
+    up with the required permissions. Taipy doesn't manage the Entra ID application.
+
+    First, you need to
+    [create an application in the Microsoft Azure portal](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal)
+    within your organization.
+    Make sure that the Redirect URI of the application is set to `http://localhost`
+    or the URI of your Taipy application.
+
+    The application needs to be [assigned permissions](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal#assign-a-role-to-the-application).
+    The required permissions are:
+
+    - `User.Read` for accessing the logged in user email from the Microsoft Graph API.
+    - `GroupMember.Read.All` for accessing the groups the user is a member of. The groups
+        are used to assign roles to the user.
+
+    From the Entra ID application, [create a new secret in the Azure portal](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal#assign-a-role-to-the-application).
+
+    The secret is only shown once, so make sure to store it in a safe place.
+    You then need to set the `ENTRA_CLIENT_SECRET` environment variable to the secret value.
+    This environment variable is required for logging in with Microsoft Entra ID accounts.
+
+    Taipy provides no support for setting up the application.
```

Comment:
What does this mean?

Reply:
We don't help people set up their Entra ID application on Azure. Maybe we should remove that line.

Reply:
I like it better. Yes. But actually, if you agree I would completely remove this line.